pkg/master/master.go

```diff
@@ -226,6 +194,16 @@ func New(c *Config) (*Master, error) {
 			enableGarbageCollection: c.EnableGarbageCollection,
 			storageFactory:          c.StorageFactory,
 		},
+
+		legacyRESTStorageProvider: corerest.LegacyRESTStorageProvider{
+			StorageFactory:            c.StorageFactory,
+			ProxyTransport:            s.ProxyTransport,
+			KubeletClient:             c.KubeletClient,
+			EventTTL:                  c.EventTTL,
+			ServiceClusterIPRange:     c.ServiceClusterIPRange,
```

Comment:
This is not clean I think. `ServiceClusterIPRange` is set via `setDefaults`. But you won't see here being set because the default are not visible outside of `GenericApiServer.New()`.